### PR TITLE
[#807] Introduction to UI docs

### LINKED ISSUE #807: Quickstart: Add UI Step: See messages & conversations in the inbox and send replies
Quickstart should have another step that shows with a gif and a description on
- How to access the UI
- How to see the conversations and incoming messages
- How to reply

### PR DESCRIPTION
**Pull request by AitorAlgorta**
resolves #807 and #966

### CODE REVIEW

File: docs/docs/apps/ui/introduction.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+</ButtonBoxList>
+
+**Overview**
+
+Inbox Screenshot/Gif
+
+<img alt="Demo Tags"
+src={useBaseUrl('img/apps/ui/demotags.gif')} />
+
+Teams & Routing Screenshot/ Gif
```
Comment (lucapette) [inline docs/docs/apps/ui/introduction.md:56]: I'd remove

File: docs/sidebars.js [hunk 2]
```diff
@@ -50,7 +50,17 @@ module.exports = {
...
     {
-      '✨ Apps': ['apps/ui'],
+      '✨ Apps': [
+        {
+          UI: [
+            'apps/ui/introduction',
+            'apps/ui/ui-quick-start',
+            'apps/ui/inbox',
+            'apps/ui/tags',
+            'apps/ui/ui-components',
```
Comment (lucapette) [inline docs/sidebars.js:60]: these `ui-` prefixes seem redundant?

**Review by lucapette — CHANGES_REQUESTED** [review]
Minor comments! But requesting changes because the teams mention as is doesn't provide good info

**Review by lucapette — DISMISSED** [review]

File: docs/docs/apps/ui/components.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
+title: UI Components
+sidebar_label: UI Components
```
Comment (chrismatix) [inline docs/docs/apps/ui/components.md:3]: So these pages will be empty right? 🤔 We could add the `draft: true` attribute.

Comment (AitorAlgorta) [reply docs/docs/apps/ui/components.md:3]: yes, they will be empty

Comment (AitorAlgorta) [reply docs/docs/apps/ui/components.md:3]: I think we want them to be there, but empty

**Review by thorstenairy — DISMISSED** [review]

**Review by pascal-airy — DISMISSED** [review]
lgtm

**Review by pascal-airy — APPROVED** [review]
good stuff!

**Pull request merged**